tests/testPhotoCal.py

```diff
@@ -114,7 +117,6 @@ def tearDown(self):
     def getAstrometrySolution(self, loglvl=Log.INFO):
         astromConfig = measAstrom.AstrometryTask.ConfigClass()
         astrom = measAstrom.AstrometryTask(config=astromConfig, refObjLoader=self.refObjLoader)
-        # use solve instead of run because the exposure has the wrong bbox
```

Comment:
Why delete this comment?  It seems somewhat helpful, and if it's not, better to update the comment than delete it, no?